decidim-core/lib/tasks/decidim_webpacker_tasks.rake

```diff
@@ -79,46 +79,27 @@ namespace :decidim do
     end
 
     def decidim_npm_packages
-      case decidim_gemspec.source
-      when Bundler::Source::Path
-        gem_path = decidim_gemspec.source.path
-        gem_path = Pathname.new(ENV["BUNDLE_GEMFILE"]).dirname.join(gem_path) if gem_path.relative?
-
-        # The packages folder needs to be copied to the application folder
-        # because the linked dependencies are not installed when packages
-        # are installed using file references outside the application root
-        # where the `package.json` is located at. For more information, see:
-        # https://github.com/npm/cli/issues/2339
-        FileUtils.cp_r("#{gem_path}/packages", rails_app_path)
-
-        {
-          dev: "./packages/dev",
-          prod: "./packages/all"
-        }
-      when Bundler::Source::Rubygems
-        if decidim_gemspec.version.to_s =~ /\.dev$/
-          # With the .dev version the package does not exist at NPM yet.
-          {
-            dev: "https://gitpkg.now.sh/decidim/decidim/packages_dev/dev",
-            prod: "https://gitpkg.now.sh/decidim/decidim/packages_dev/all"
-          }
-        else
-          {
-            dev: "@decidim/dev@~#{decidim_gemspec.version}",
-            prod: "@decidim/all@~#{decidim_gemspec.version}"
-          }
-        end
-      when Bundler::Source::Git
-        github_repo =
-          decidim_gemspec.source.uri[%r{github\.com/([^/]*/[^.]*)}, 1] ||
-          "decidim/decidim"
-        branch = decidim_gemspec.source.branch
-
-        {
-          dev: "https://gitpkg.now.sh/#{github_repo}/packages_dev/dev?#{branch}",
-          prod: "https://gitpkg.now.sh/#{github_repo}/packages_dev/all?#{branch}"
+      if decidim_gemspec.source.is_a?(Bundler::Source::Rubygems)
```

Comment:
With this change I'm trying to simplify things as much as possible. My idea here is: if you are using a gem, there was a release and there should be a released npm package with the exactly same version, it doesn't matter the type of version.

Reply:
@leio10 Yes, isn't it already so? If the gem source is Rubygems (i.e. a released gem), it will refer to the matching NPM packages.

Or did I get something wrong here?

Reply:
I'm referring that [we are now checking for the `.dev` suffix to use `gitpkg` in that case](https://github.com/decidim/decidim/blob/a2c340b821dce21eead0b934ba80d37c3ca473ad/decidim-core/lib/tasks/decidim_webpacker_tasks.rake#L99). I'd like to remove that part if possible.

Reply:
Ah, that is because the specs that are testing building the gems and installing them locally. In those situations, the source is reported to be "Rubygems" but the NPM packages for those versions are not yet available.

Reply:
Ah, ok, I'm slowly understanding all the issues, thanks! ❤️